apps/main/src/components/PageCreatePool/TokensInPool/index.tsx

```diff
@@ -113,6 +103,23 @@ const TokensInPool = ({ curve, chainId, haveSigner }: Props) => {
   const handleInpChange = useCallback(
     (name: TokenId, value: string) => {
       if (!value.startsWith('0x')) return
+
+      let basePoolCoins: string[] = []
```

Comment:
this is a lot of logic for a callback inside a component. Could we extract it to a function?